securedrop/db.py

```diff
@@ -269,17 +274,20 @@ def _scrypt_hash(self, password, salt, params=None):
 
     MAX_PASSWORD_LEN = 128
 
-    def set_password(self, password):
+    @staticmethod
+    def check_password_acceptable(password):
         # Enforce a reasonable maximum length for passwords to avoid DoS
-        if len(password) > self.MAX_PASSWORD_LEN:
+        if len(password) > Journalist.MAX_PASSWORD_LEN:
```

Comment:
> I think this does need to be static so it can be preemptively used without creating a Journalist instance.

You can do the same thing with a class method. The main advantage in this case is that, by writing `cls.MAX_PASSWORD_LEN`, even if the `Journalist` class where to change names, the method would still work w/o requiring you to replace `Journalist` in the method body with the new class name. In this case it's unlikely we'll change the name of the `Journalist` class, but I still think it's good practice. This SO answer http://stackoverflow.com/a/12179752 might give a little more info. on when it's better to use one or the other.